RACK-Ontology/ontology/SYSTEM.sadl

```diff
@@ -10,67 +10,68 @@
+	provides (note "List of entities that this SYSTEM provides to others") describes SYSTEM with values of type FUNCTION.
 
-	requires (note "List of entities that this SYSTEM requires from others") describes SYSTEM with values of type ENTITY.
-	requires is a type of wasDerivedFrom.
+	requires (note "List of entities that this SYSTEM requires from others") describes SYSTEM with values of type FUNCTION.
 
-	function (note "functionality / capability of a SYSTEM") describes SYSTEM with values of type FUNCTION. 
-	function is a type of wasDerivedFrom. 
+	functionalityOverview (note "functionality / capability of a SYSTEM") describes SYSTEM with values of type string. 
+	functionalityOverview describes SYSTEM with at most 1 value.
 	
 INTERFACE
-	(note "INTERFACEs describe directed connections between an ENTITY (e.g. SYSTEM) providing what another ENTITY (e.g. SYSTEM) requires")
+	(note "INTERFACEs describe directed connections between a SYSTEM providing what another SYSTEM requires")
 	is a type of ENTITY.
 
-	source (note "ENTITY (e.g. SYSTEM) that provides what the INTERFACE offers") describes INTERFACE with values of type ENTITY.
-	source is the inverse of provides.
-	source is a type of wasDerivedFrom.
+	commodity (note "the thing conveyed by the interface") describes INTERFACE with values of type string.
+	commodity describes INTERFACE with at most 1 value.
 
-	destination (note "ENTITY (e.g. SYSTEM) that requires what the INTERFACE offers") describes INTERFACE with values of type ENTITY.
-	destination is the inverse of requires.
-	destination is a type of wasDerivedFrom.
+	source (note "SYSTEM that provides what the INTERFACE offers") describes INTERFACE with values of type SYSTEM.
+	// source is the inverse of sink.
+	source is a type of wasImpactedBy.
+
+	destination (note "SYSTEM that requires what the INTERFACE offers") describes INTERFACE with values of type SYSTEM.
+	// sink is the inverse of source.
+	destination is a type of wasImpactedBy.
 
 	identifiedBy (note "ACTIVITY out of which this INTERFACE was identified") describes INTERFACE with values of type ACTIVITY.
-	identifiedBy is a type of wasGeneratedBy.
 
 
 SYSTEM_DEVELOPMENT
 	(note "ACTIVITY relating to the creation of one or more some SYSTEMs")
 	is a type of ACTIVITY.
 
 	developedBy (note "Organization that is responsible for producing a system") describes SYSTEM_DEVELOPMENT with values of type AGENT.
+	developedBy is a type of wasAssociatedWith.
 
 FUNCTION  
 	(note "defines functionality / capability")
 	is a type of ENTITY.
 
-	// If we introduce "parent" property for THING, then we could drop parentFunction.	
-	parentFunction (note "allows for decomposition of functionality") describes FUNCTION with a single value of type FUNCTION.
-	parentFunction is a type of wasDerivedFrom.
+   parentFunction (note "allows for decomposition of functionality") describes FUNCTION with a single value of type FUNCTION.
 
-// Later we could add more structure to OP_ENV.
 OP_ENV   
```

Comment:
Seems fine to expand naming to be consistent. Let's let the data model decision team figure out if some things should be pushed to an ontology specific to an individual TA1. I'd rather us stop trying to figure out what should be or shouldn't be in the ontology once we have TA1s engaged, since they have the direct need. What do you think?